### ISSUE TITLE
[Suggestion] Trade With Seller for outgoing panel

### ISSUE THREAD

**Issue by videnfost** [labo89/TraderForPoe#33]
I LOVE this program for trading in PoE. I experimented with this, MercuryTrade, and PoE Trades Companion, but I prefer this for sure. However, this program is missing one thing (in my opinion). The outgoing panel has shortcuts for everything you need to do during a trade, except for initiating a trade with a seller. I understand some people view the buyer initiating a trade as a bad thing, but I think it is personal preference and a lot of people enjoy when a buyer initiates. I think it would be nice to have the option to choose a trade initiation shortcut.

**Comment by mangni** [comment]
Yes please add this.  Layout should be the same for incoming and outgoing panel.

**Comment by g0sy** [comment]
Please add the outgoing trade panel, its kinda hard to keep track without it
